Client/Client.CLI/App.cpp

```diff
@@ -1,5 +1,5 @@
 #include "App.hpp"
-#include "Utility/YasSerializer.hpp"
+#include "Network/YasSerializer.hpp"
```

Comment:
Why did you move serializatio, compression, encryption handlers from utility to network?

Reply:
Because it's using namespace Network.